cmake/std/atdm/sems-rhel7/environment.sh

```diff
@@ -190,7 +190,6 @@ elif [[ "$ATDM_CONFIG_COMPILER" == "CUDA-9.2" ]] ; then
   export LAPACK_ROOT=/usr/lib64/atlas
   export ATDM_CONFIG_LAPACK_LIBS="-L${LAPACK_ROOT};-llapack"
   export ATDM_CONFIG_BLAS_LIBS="-L${BLAS_ROOT}/lib;-lblas"
-  # Needed for some Tpetra UVM stuff
```

Comment:
That was the case the last time I checked.  I had to add this back specifically to make a bunch of Tpetra tests pass.  When did Tpetra remove this requirement?

If it is true that Tpetra does not need this anymore, then we should just remove this and test to make sure.

Reply:
@bartlettroscoe Most of _Tpetra_ runs fine without `CUDA_LAUNCH_BLOCKING=1`, but most downstream packages do _not_ run fine.  I merely object to the characterization of Tpetra as the source of all trouble ;-)  @kddevin doesn't need more stress in her life....

Reply:
Then let's change the comment to something like "some Trilinos tests require this to run correctly"

Reply:
Done; thanks!